iib/workers/tasks/utils.py

```diff
@@ -634,18 +637,24 @@ def run_cmd(cmd, params=None, exc_msg=None, strict=True):
     params.setdefault('stdout', subprocess.PIPE)
 
     log.debug('Running the command "%s"', ' '.join(cmd))
-    response = subprocess.run(cmd, **params)
+    response: subprocess.CompletedProcess = subprocess.run(cmd, **params)
 
     if strict and response.returncode != 0:
         log.error('The command "%s" failed with: %s', ' '.join(cmd), response.stderr)
         if cmd[0] == 'opm':
             # Capture the error message right before the help display
-            regex = r'^(?:Error: )(.+)$'
+            regex: str = r'^(?:Error: )(.+)$'
             # Start from the last log message since the failure occurs near the bottom
             for msg in reversed(response.stderr.splitlines()):
-                match = re.match(regex, msg)
+                match: re.Match = re.match(regex, msg)
                 if match:
                     raise IIBError(f'{exc_msg.rstrip(".")}: {match.groups()[0]}')
+        if cmd[0] == 'buildah':
+            # Check for HTTP 50X errors on buildah
+            regex: str = r'(error creating build container).*(HTTP status: 50[0-9]) .*'
```

Comment:
This might not catch everything, have a look on this error msg: 
```ERROR utils.run_cmd The command "buildah bud --no-cache --override-arch ppc64le --arch ppc64le -t iib-build:55156-ppc64le -f /tmp/iib-cyojeb6s/index.Dockerfile" failed with: Trying to pull *******
error creating build container: parsing image configuration: Error fetching blob: invalid status code from registry 503 (Service Unavailable)
time="2021-12-08T00:09:54-05:00" level=error msg="exit status 125"```

Reply:
Thanks!

I'll change it to this regex and push again:

```
(error creating build container).*(50[0-9]) .*
```

Reply:
Pushed, I tested this regex against your file and the error on JIRA, please let me know if it's good :)